### DISCUSSION TITLE
Play again sound when clicking again on a tile

### DISCUSSION THREAD

**Discussion by emmanuelrouxfr** [bamdadfr/memaudio-org--www#300] [General]
Once a tile has been selected its associated sound is played.
It would be nice to be able to play the sound again by clicking on the same tile (sometimes there can be noise around the player and it may be nice to repeat the sound once the noise has gone away ...).

**Comment by bamdadfr** [comment]
Interesting suggestion that is definitely doable.

I am concerned about the sound stacking effect though.

For instance, how should the app behave when a selected tile is spam clicked?

Please discuss it [here](https://github.com/bamdadsabbagh/memaudio-org--www/issues/186)
